src/main/java/org/kohsuke/github/GHObject.java

```diff
@@ -79,36 +78,19 @@ public Map<String, List<String>> getResponseHeaderFields() {
      * @throws IOException
      *             on error
      */
-    @WithBridgeMethods(value = String.class, adapterMethod = "createdAtStr")
     public Date getCreatedAt() throws IOException {
         return GitHubClient.parseDate(createdAt);
     }
 
-    @SuppressFBWarnings(value = "UPM_UNCALLED_PRIVATE_METHOD", justification = "Bridge method of getCreatedAt")
-    private Object createdAtStr(Date id, Class type) {
-        return createdAt;
-    }
-
     /**
      * Gets url.
      *
      * @return API URL of this object.
      */
-    @WithBridgeMethods(value = String.class, adapterMethod = "urlToString")
     public URL getUrl() {
         return GitHubClient.parseURL(url);
     }
 
-    /**
-     * Gets html url.
-     *
-     * @return URL of this object for humans, which renders some HTML.
-     * @throws IOException
-     *             on error
-     */
-    @WithBridgeMethods(value = String.class, adapterMethod = "urlToString")
-    public abstract URL getHtmlUrl() throws IOException;
```

Comment:
TODO: verify this is the right thing to do.   Not deprecated... 

Maybe just add the field and implement this.  Unclear why this was abstract.

Reply:
Decided to go ahead with this removal. If it breaks people this can be rolled back as a non-abstract method that returns `null`.